community/data-services.en.md

```diff
@@ -42,37 +38,21 @@ Monthly exports of the RISM data is available for download in MARCXML format on
+RISM data is available in several linked data contexts. Wikidata provides a RISM ID property, [P5504](https://www.wikidata.org/wiki/Property:P5504){:target="_blank"}, that may be used in Wikidata-oriented SPARQL queries. There is also a RISM Siglum property, [P11550](https://www.wikidata.org/wiki/Property:P11550), that can be used as identifiers for institutions that have an assigned [RISM Siglum](https://rism.info/community/sigla.html).
 
-RISM Data is available in several Linked Data contexts. Wikidata provides a RISM ID property, [P5504](https://www.wikidata.org/wiki/Property:P5504){:target="_blank"}, that may be used in Wikidata-oriented SPARQL queries. There is also a RISM Siglum property, [P11550](https://www.wikidata.org/wiki/Property:P11550), that can be used as identifiers for institutions that have an assigned [RISM Siglum](https://rism.info/community/sigla.html).
-
-We have a standard prefix and compact identifier, `rism:`, registered with the [Identifiers.org Resolution Service](https://identifiers.org){:target="_blank"}. This can also be used to resolve RISM Identifiers to records in the systems that support these types of identifiers.
+We have a standard prefix and compact identifier, `rism:`, registered with the [Identifiers.org Resolution Service](https://identifiers.org){:target="_blank"}. This can also be used to resolve RISM identifiers to records in the systems that support these types of identifiers.
 
 For example:
 
  - Identifiers.org: [http://identifiers.org/rism:people/30002781](http://identifiers.org/rism:people/30002781){:target="_blank"} (Saint-Georges, Joseph Bologne de (1745-1799))
  - n2t.net: [https://n2t.net/rism:institutions/30078247](https://n2t.net/rism:institutions/30078247){:target="_blank"} (National Library of Ukraine, Music Department)
 
-The RISM Catalog provides the [RISM data in Turtle format](https://opac.rism.info/main-menu-/kachelmenu/data){:target="_blank"} for download and import into Linked Data systems. Note that this data does not follow the RISM Identifiers scheme described above.
+The RISM Catalog provides the [RISM data in Turtle format](https://opac.rism.info/main-menu-/kachelmenu/data){:target="_blank"} for download and import into linked data systems. Note that this data does not follow the RISM identifiers scheme described above.
 
 ## RISM Online API
 
-RISM Online has several web-based APIs (Application Programming Interface) available for public use. Details on how to use these APIs are available in the [RISM Online Documentation](https://rism.online/docs/){:target="_blank"}. For most users this should be the preferred way of accessing and using the RISM Data.
+RISM Online has several web-based APIs (Application Programming Interface) available for public use. Details on how to use these APIs are available in the [RISM Online Documentation](https://rism.online/docs/){:target="_blank"}. For most users this should be the preferred way of accessing and using the RISM data.
```

Comment:
@ahankinson Maybe provide clarification for who the target "most users" are? I'm not sure if the people I'm in contact with would know how to use the API.